test/goldens/end2end/__timestamp.txt

```diff
@@ -1 +1 @@
-2021-05-21T07:12:03.731197Z
+2021-05-26T01:34:20.962183Z
```

Comment:
Note: end2end tests should be able to handle new versions now, if you don't change the output, PRs should not affect these goldens. We are using this timestamp to cut out newer versions.